internal/pkg/template/templates/workloads/services/backend/manifest.yml

```diff
@@ -43,10 +43,11 @@ count: {{.Count.Value}}       # Number of tasks that should be running in your s
 {{- if not .TaskConfig.IsWindows }}
 exec: true     # Enable running commands in your container.
 {{- end}}
-
-storage:
+  
 {{- if not .TaskConfig.IsWindows}}
-  readonly_fs: true       # Limit to read-only access to mounted root filesystems by default.
+
+# storage:
+  # readonly_fs: true       # Limit to read-only access to mounted root filesystems.
```

Comment:
So by default `readonly_fs` won't be enabled?  But isn't that contradicting with the original [ask](https://github.com/aws/copilot-cli/issues/4010)?

Reply:
Adding that field by default in manifest adds to confusion for users with `RUN some write command` in their Dockerfile. Worse, they'd have to wait for the circuit breaker to get triggered in order to know the deployment fails.

This is a middle ground between "adding it by default" and "not adding it at all"

Reply:
> Worse, they'd have to wait for the circuit breaker to get triggered in order to know the deployment fails.
Totally agree to this.